common/model/label-field.js

```diff
@@ -1,7 +1,9 @@
 // @flow
 import type {HTMLString} from '../services/prismic/types';
+import type {ContentFormatId} from './content-format-id';
 
 export type LabelField = {|
+  id: ?ContentFormatId,
```

Comment:
I'd probably leave this as a string, as it will always be a string, and we might get some we aren't sure of, but validate later on against it like you did.